src/query-builder/select-query-builder.ts

```diff
@@ -1500,6 +1500,26 @@ export interface SelectQueryBuilder<DB, TB extends keyof DB, O>
    */
   clearOrderBy(): SelectQueryBuilder<DB, TB, O>
 
+  /**
+   * Clears `group by` clause from the query.
+   *
+   * ### Examples
+   *
+   * ```ts
+   * db.selectFrom('person')
+   *   .selectAll()
+   *   .orderBy('id')
+   *   .clearGroupBy()
```

Comment:
The example doesn't make sense. There's no `.groupBy` to clear.

Reply:
Good catch! I didn't even notice 😅